exp/docgen/www/template/function.php

```diff
@@ -22,41 +45,94 @@
 if (isset($PageFunction['class_name'])) {
 ?>
     <h1 class="page-header"><?php echo htmlspecialchars($FunctionName); ?> Method</h1>
+<?php } else if ($PageFunction['kind'] === 'forward') { ?>
+    <h1 class="page-header"><?php echo htmlspecialchars($FunctionName); ?> Forward</h1>
 <?php } else { ?>
     <h1 class="page-header"><?php echo htmlspecialchars($FunctionName); ?> Function</h1>
 <?php } ?>
 
-<h4 class="sub-header2">Syntax</h4>
-<pre class="syntax"><?php echo htmlspecialchars($PageFunction['signature']); ?></pre>
-
-<?php if (!empty($Parameters)): ?>
-<h4 class="sub-header2">Usage</h4>
-<div class="table-responsive">
-    <table class="table table-condensed table-bordered">
-        <?php
-            foreach ($Parameters as $Param) {
-                echo '<tr>' .
-                     '<td width="15%"><pre class="description">' . htmlspecialchars($Param['name']) . '</pre></td>' .
-                     '<td width="15%"><pre class="description">' . htmlspecialchars($Param['type']) . '</pre></td>' .
-                     '<td width="70%">' . htmlspecialchars($Param['doc']) . '</td>' .
-                     '</tr>';
-            }
-        ?>
-    </table>
-</div>
-<?php endif; ?>
-
-<h4 class="sub-header2">Description</h4>
-<?php RenderDescription($PageFunction['brief']); ?>
+<?php if (empty($PageFunction['brief'])) { ?>
+  <p class="text-muted">This function has no description.</p>
+<?php } else { RenderDescription($PageFunction['brief']); } ?>
+
+<?php
+if (isset($Data['return'])) {
+    $Type = NormalizeType($Data['return']['type']);
```

Comment:
This is something that should be cross-indexed in the database itself, but that can wait for an eventual docgen rewrite.